apps/frontend/src/store/previewTree/previewTreeSlice.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const previewTreeSlice = createSlice({
+    name: 'previewTree',
+    initialState: {
+        value: ['ded']
+    },
+    reducers: {
```

Comment:
I thought your configuration would be more...working.

Reply:
What??
It's just a mock data